### PR TITLE
added name

### PR DESCRIPTION
**Pull request by DarrianMC**
added name

### CODE REVIEW

**Comment by kbuffardi** [comment]
There's two different files that have been made for the team. Please put all names/usernames in one file and remove the other file from the repo

File: BudgetTravel.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
```
Comment (kbuffardi) [inline BudgetTravel.md:1]: This file is unnecessary. Please remove it (git rm)

File: README.md [hunk 2]
```diff
@@ -4,6 +4,7 @@
 
 * [BilliardsTrainer](BilliardsTrainer.md)
 * [VidDiffusion](VidDiffusion.md)
+* [BudgetTravel](budgettravel.md)
```
Comment (kbuffardi) [inline README.md:7]: This list is supposed to be in alphabetic order.

**Review by kbuffardi — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (RioSakao) [reply README.md:6]: Fixed.

File: budgettravel.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# budgettravel
+## Team Members
```
Comment (kbuffardi) [inline budgettravel.md:2]: There needs to be a blank line between the first header and the second for the markdown to render properly.

**Review by kbuffardi — CHANGES_REQUESTED** [review]

**Comment by DarrianMC** [comment]
Ok, I just fixed it.

**Pull request merged**